appendix/backup-and-restore/docker-compose.rst

```diff
@@ -0,0 +1,151 @@
+Backup & Restore (Docker)
+=========================
+
+This section shows some basics about the backup and restore process for a docker
+compose based deployment of Zammad. If you already use a different method or
+tool, it is fine to stick with it as long as it works. In such a case, consider
+using the :ref:`disable-backup-service scenario <additional-scenarios>`.
+
+If you're familiar with docker, the Quick Start section below includes the
+information you'll need. The :ref:`file-handling` section covers some examples
+about how to handle the backup files and to copy it into a docker volume to
+restore it.
+
+Quick Start
+-----------
+
+Backup
+^^^^^^
+
+By default, a backup is created at each start of the stack as well as at 3
+o'clock each night. The backup is stored in the volume of the
+**zammad-backup** container under ``/var/tmp/zammad``.
+
+Restore
+^^^^^^^
+
+.. warning::
+   Be aware that the restore process always uses the latest backup according
+   to the timestamp of the file name.
+
+#. Start the new stack at least once so a Zammad database is available.
+#. Copy or move the backup files to ``/var/tmp/zammad/restore/`` inside the
+   volume of the **zammad-backup** container.
+#. Stop the stack.
+#. Start the stack. The restore process is triggered if the ``restore``
+   directory is detected and the backup files are in place.
+#. After the restore process has finished, the ``restore`` directory got renamed.
+   You can safely delete it now.
+
+.. _file-handling:
+
+File Handling
+-------------
+
+If you're not sure how to handle the backup files and how to create the
+``restore`` directory in the docker volume, you can find some examples below.
+
+Restore Inside the Same Stack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Requires:** console access to the zammad-backup container.
+
+If you want to restore a backup from the same stack, you just have to create
+the directory and copy/move the files into it. The following example starts the
+**zammad-backup** container and drops the unused one, gets access to the console
```

Comment:
`drops the unused one, gets access to the console and` - I'm not sure what you mean by this part. This could be deleted.
Nothing gets dropped AFAIR, even if zammad-backup is already running it should keep running.